### ISSUE TITLE
wivar not work

### ISSUE THREAD

**Issue by onmyway133** [facebook/chisel#106]
In my ViewController, I have

```
@interface ViewController ()
@property (nonatomic, strong) NSNumber *number;
@end
```

And in LLDB
`wivar self _number`

I change that number, but the program does not pause. Use Xcode Debugger -> Right click on variable  -> Watch works

**Comment by kastiglione** [comment]
I missed this issue, sorry about that! I'll look into this.

**Comment by idrisr** [comment]
I can confirm this bug. Here is a simple app to quickly reproduce the issue: https://github.com/idrisr/wivarBug

**Comment by idrisr** [comment]
I've submited this pull request to fix this issue: https://github.com/facebook/chisel/pull/117

**Comment by kastiglione** [comment]
Fixed by #117
